account_brand/views/account_invoice.xml

```diff
@@ -0,0 +1,17 @@
+<odoo>
+
+    <!-- Copyright (C) 2019 Open Source Integrators
+         License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+    <record id="invoice_form_brand" model="ir.ui.view">
```

Comment:
Please align with the [desired standard for view naming](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action)